apps/ledger-live-mobile/src/components/FirebaseFeatureFlags.tsx

```diff
@@ -172,12 +172,14 @@ export const FirebaseFeatureFlagsProvider: React.FC<Props> = ({ children }) => {
 
   return (
     <FeatureFlagsProvider
-      isFeature={isFeature}
-      getFeature={wrappedGetFeature}
-      overrideFeature={overrideFeature}
-      resetFeature={resetFeature}
-      resetFeatures={resetFeatures}
-      getAllFlags={getAllFlags}
+      value={{
+        isFeature,
+        getFeature: wrappedGetFeature,
+        overrideFeature,
+        resetFeature,
+        resetFeatures,
+        getAllFlags,
+      }}
```

Comment:
`FeatureFlagsProvider` is now defined as `FeatureFlagsContext.Provider` so that the context value passed to it isn't destructured into a new object. (cf. definition of `FeatureFlagsProvider` in live-common).
This destructuration into a new object became obvious when writing a unit test for the provider. There were no important consequences to that but it's better to create this new object explicitly. Now the context value passed to it has to be defined directly in the component that renders it.

Reply:
should it be `useMemo` ied tho? like isn't it going to re-render all consumer of this each time the component itself render() ?

Reply:
done @gre 👍